server/src/WebsocketServer.hs

```diff
@@ -96,29 +87,58 @@ authorizePendingConnection core conn
 handleClient :: WS.Connection -> Path -> Core -> IO ()
 handleClient conn path core = do
   uuid <- newUUID
+  -- TODO: Add a config field to retrieve the length for this queue.
+  subscriberState <- newSubscriberState 100
```

Comment:
With this TODO being fixed this looks like a viable option. I'd probably use a smaller value here though, we expect subscribers to be fast. If they aren't, they deserve losing the updates.